app/views/tables/_table.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="table_btn_list">
+  <% if table.available? %>
```

Comment:
cho toàn bộ if; else vào hepler để check xem nó là btn-success,  btn-info hay btn-default rồi gọi ra ở đây